caffe2/CMakeLists.txt

```diff
@@ -779,6 +779,7 @@ endif()
 
 if(NOT BUILD_LIBTORCHLESS)
 add_library(torch_cpu ${Caffe2_CPU_SRCS})
+# target_compile_definitions(torch_cpu PRIVATE -DEXPORT_AOTI_FUNCTIONS)
```

Comment:
I guess all the changes here and in the file below are leftovers and should be removed?

Reply:
@albanD sure, this PR was reverted many times, I'm not sure how to pass meta internal CI, so I keep the old code here.
I will clean up code certainly landed.